pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [build-system]
-requires = ["setuptools"]  # the build tool to use
+requires = ["setuptools","setuptools_scm>=8"]  # the build tool to use
 build-backend = "setuptools.build_meta"  # the function to use to build the package
 
 [project]
 name = "pymecht"
-version = "0.9"
+#version = "0.9"
+dynamic = ["version"]
 description = "This is PYthon-based repository is for MECHanics of Tissue mechanics. The focus is on flexibility of adding new constitutive models and varying their parameters."
 readme = "README.md"
 authors = [{ name = "Ankush Aggarwal", email = "ankush.aggarwal@glasgow.ac.uk" }, { name = "Ross Williams", email = "r.williams.2@research.gla.ac.uk" }]
```

Comment:
Just noticed that my email address should probably be updated here (ross.williams@glasgow.ac.uk). You might want to include that as a separate commit in this pull request since it doesn't really merit a separate one.

Reply:
Thanks! Updated it.